config/python/pipeline.yaml

```diff
@@ -50,6 +50,7 @@ spec:
         - name: output
           workspace: git-source
     - name: install-dependencies
+      retries: 2
```

Comment:
Are you seeing issues there, or is this leftover?

Reply:
left over from the testing, ill bump it down to 1